src/main/java/org/springframework/shell/Bootstrap.java

```diff
@@ -37,25 +40,26 @@
  * @author Ben Alex (original Roo code)
  * @author Mark Pollack
  * @author David Winterfeldt
+ * @author Robin Howlett
  * 
  */
 public class Bootstrap {
 
+	public final static String[] NO_BASE_PACKAGES = null;
+	
 	private final static String[] CONTEXT_PATH = { "classpath*:/META-INF/spring/spring-shell-plugin.xml" };
 
 	private static Bootstrap bootstrap;
-
-	private static StopWatch sw = new StopWatch("Spring Shell");
-
 	private static CommandLine commandLine;
+	private static StopWatch sw = new StopWatch("Spring Shell");
 
 	private GenericApplicationContext ctx;
 
-	public static void main(String[] args) throws IOException {
+	public static void main(String[] args, String... basePackages) throws IOException {
```

Comment:
Thanks for catching that mistake. I've updated to PR to revert back to a spec-compliant main method and to use a default package to scan (similar to how XML config is supported)